spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java

```diff
@@ -190,45 +195,51 @@ public void shutdown() throws Exception {
 		}
 	}
 
+	/**
+	 * Will check if {@link LocalDeployerProperties#INHERIT_LOGGING} is set by
+	 * checking deployment properties and then application properties
+	 */
+	private boolean shouldInheritLogging(AppDeploymentRequest request){
+		boolean logToConsole = false;
+		if (request.getDeploymentProperties().containsKey(LocalDeployerProperties.INHERIT_LOGGING)){
+			logToConsole = Boolean.parseBoolean(request.getDeploymentProperties().get(LocalDeployerProperties.INHERIT_LOGGING));
+		}
+		else if (request.getDefinition().getProperties().containsKey(LocalDeployerProperties.INHERIT_LOGGING)){
```

Comment:
I think you are ;)
Per application deployment property is set via 
```
stream deploy foo --properties `deployer.<appname>.<propname>=<value>
```

Application vs Deployment property choice is really that: does this property make sense out of SCDF, if the app was used standalone / without a deployer? If yes, then it may be a case of the former.

Please note that I haven't properly reviewed the PR, so maybe I don't get your "per app / per deployer" split, but to me, this should definitely not be an _application property_ (definition).